ui/utilities/media-objects/flavors/base/_index.scss

```diff
@@ -62,7 +62,7 @@
     }
 
     &.slds-media--small .slds-media__figure {
-      margin-left: $spacing-xx-small;
+      margin-#{$left}: $spacing-xx-small;
```

Comment:
Our use case is a client based in Israel so for us loading a different css is fine, but I understand there's other use cases to cater for others, so I'd rather get something that everyone can use (including us) and we're happy to do this for the community.

Is it better to detect rtl off the `[dir=rtl]` attribute on a parent element instead of adding an `slds-rtl` class?